Makefile

```diff
@@ -1,107 +1,113 @@
 #/bin/bash
 
-BASEDIR = $(shell pwd)
-SOURCE = "${BASEDIR}/lib"
-IMAGE = "elastica"
+.PHONY: update clean build setup start stop destroy run checkstyle checkstyle-ci code-browser cpd messdetector messdetector-ci dependencies phpunit test tests doc lint syntax-check loc phploc gource 
```

Comment:
@im-denisenko Here is the pull request for the Makefile. Have a look and merge it if you think that helps: https://github.com/ruflin/Elastica/pull/917